api/infrastructure/parallelcluster-api.yaml

```diff
@@ -81,6 +81,8 @@ Parameters:
       When specified, IAM permissions assumed by the API are conditionally restricted to the usage of the given PermissionsBoundary
     Type: String
     Default: ''
+    ConstraintDescription: "The value must be a valid ARN in the format: arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/PolicyNameParameter"
```

Comment:
We should not enforce any specific partition or account id. You should replace the partition and account variables with a wildcard `.*`